packages/core/src/runtime/resolveStyleRules.ts

```diff
@@ -54,6 +55,14 @@ export function resolveStyleRules(
 ): [CSSClassesMap, CSSRulesByBucket] {
   // eslint-disable-next-line guard-for-in
   for (const property in styles) {
+    // eslint-disable-next-line no-prototype-builtins
+    if (process.env.NODE_ENV !== 'production' && UNSUPPORTED_CSS_PROPERTIES.hasOwnProperty(property)) {
+      if (process.env.NODE_ENV !== 'test') {
+        console.error('Unsupported CSS property', property, 'will not be resolved');
+      }
+      continue;
```

Comment:
I think that stripping the unsupported CSS properties only in dev mode is a bad idea - that will lead to inconsistent CSS between dev and prod.

The final CSS should be the same in all three modes - dev, prod and babel.
We can conditionally warn or throw but not strip.

Reply:
That's a fair point, I think that the code to do this (if I'm doing it correctly) shouldn't have a large bundle size impact on the library, which was the original motivation in the issue to do it only at dev time.

What is your appetite to strip unsupported css in all modes ?

Reply:
> What is your appetite to strip unsupported css in all modes ?

Works for me.

Reply:
> I think that stripping the unsupported CSS properties only in dev mode is a bad idea - that will lead to inconsistent CSS between dev and prod.

If we will keep this in production it will increase bundle size. It's the original motivation to have it only in `devMode`.

Reply:
I've added a new runtime bundle size fixture to this PR: here are the results with/without production check

with production check:

![image](https://user-images.githubusercontent.com/20744592/155310673-bbf468d2-db25-4e1b-852f-0645357a6ff7.png)


without production check:

![image](https://user-images.githubusercontent.com/20744592/155310689-c033c12f-c9d4-453a-9b93-6cb3e113b647.png)

Reply:
I've removed the production check so that we're consistently stripping out the unsupported styes. I've kept the bundle size fixture